src/org/loklak/harvester/Post.java

```diff
@@ -35,8 +37,16 @@ public void setTimestamp() {
         this.setTimestamp(timestamp);
     }
 
-    //public abstract void getPostId();
+    public Date getTimestampDate() {
+        return new Date(this.timestamp);
+    }
+
+    //TODO: Set up TwitterTweet before setting this as abstract
+    public void setPostId() { }
```

Comment:
https://dzone.com/articles/getter-setter-use-or-not-use

go through the TODO mentioned, it is to be set abstract.